README.md

```diff
@@ -168,7 +168,7 @@ let expected = "+------+----------------+---------------+\n\
 assert_eq!(table, expected);
 ```
 
-The next example shows a builder example.
+The next example illustrates a builder method.
```

Comment:
what `method`?

there is:
```rs
let mut builder = Builder::default();
```
and
```rs
builder.push_record(words);
```

to me it's more of a `builder` example than a `builder` `method` illustration

Reply:
`Builder` used here as a `pattern`.
So maybe pattern is a better alternative here?

Reply:
agreed